alan.json

```
@@ -1,42 +1,71 @@
+  "The name 'Alan' does not contain any numbers, only letters.",
+  "Alan Brouilette has a policy of ordering unfamiliar menu items at restaurants.",
+  "Max size for a male Timber Rattlesnake: 4.5 feet. #SnakeFacts",
+  "Alan Dowa Dolma is a female Chinese singer of Tibetan ethnicity, active in both China and Japan",
+  "Alan IV, Duke of Brittany, 1063 - 1119, was also known as Alan Fergant ('Iron Glove' in Breton language)",
+  "Alan Moore wrote _Watchmen, V for Vendetta_, and 'The Killing Joke', and is wildly underrepresented in these facts.",
+  "Ferris Bueller's closest friend, Cameron Frye, was portrayed by actor Alan Ruck.",
+  "Víctor Manuel Soto Flores is a Mexican Luchador (professional wrestler) currently working under the ring name Alan and also under a mask as the enmascarado Drago.",
+  "Alain de la Roche, or Alanus de Rupe, or Blessed Alan, was a canonized saint, 'Alan of the Rock'.",
+  "https://frinkiac.com/meme/S06E11/403335/m/SGkuIEknbSBBbGFuLiBJJ20geW91cgogY29waWxvdC4=",
+  "Alan Atlas is so agile he once dodged a punch from Mike Tyson.",
+  ":tudyk: :tudyk: :tudyk: I am Alan Tudyk and I have returned. I have unfinished business. Where is @randy? :tudyk: :tudyk: :tudyk:",
+  "Alan Atlas was supposed to be named Isaac after his great-grandfather, but his mother thought Isaac was too old-fashioned of a name so she substituted Alan for Isaac.",
+  "Alan L. Hodgkin shared the 1963 Nobel Prize in Physiology or Medicine for 'discoveries concerning the ionic mechanisms involved in excitation and inhibition in the peripheral and central portions of the nerve cell membrane'.",
+  "The Alan Parsons Project instrumental 'Sirius' is the music bed underlying this state-of-the-art CGI portion of the Chicago Bulls' late-90's player introduction sequence: https://www.youtube.com/watch?v=zmFkbxUkGo4",
+  "The name 'Alan' is contained in the word 'Phalanx.' This is not unintentional.",
+  "Alan May was a fan favorite in his playing days as a Washington Capital - and is now an analyst for Caps games on Comcast Sportsnet.",
+  "The #alan Slack channel was accidentally founded by @andrewmaier on June 22, 2015.",
+  "When Emileigh Barnes was in kindergarten, she lived in Mississippi and her best friend was Alan Kuhnle. They used to climb magnolia trees and ponder whether God would save them if they jumped out of them.",
+  "Alan Smithee is an official pseudonym used by film directors who wish to disown a project, coined in 1968.",
+  "Alan Sherman once sang about Alan deLevie's favorite beverage, seltzer.",
+  "Whenever @mgwalker hears 'seltzer water', he thinks of that song by Alan Sherman.",
+  "Incorrect spellings of Alan include 'Alen', 'Allen', 'Alyn', 'Allyn', and 'Allan'.",
+  "Alan Atlas sometimes spells his name 'Al3an'. The '3' is silent,"
```

idk if the comma inside is intentional?

```suggestion
  "Alan Atlas sometimes spells his name 'Al3an'. The '3' is silent."
```